version/src/main/java/org/wildfly/jdk/version/JdkType.java

```diff
@@ -0,0 +1,170 @@
+/*
```

Comment:
WFCORE "server" maven subproject cannot see WFCORE "host-controller" subproject.
I didn't want to introduce "server" -> "host-controller" dependency, so
I analyzed "host-controller", "process-controller" and "server" maven subproject dependencies
and only "version" is available to all of them. What other location would you suggest @jamezp and @bstansberry ?

Reply:
I don't see any use of this in the 'server' module. My thought was you did this because you used it in 'process-controller' but I believe you are removing that.

If it's needed in 'server' it could go in server as 'host-controller' already depends on 'server'.

Reply:
I moved it to "process-controller" module @bstansberry